platform/android/detect.py

```diff
@@ -134,7 +134,7 @@ def mySpawn(sh, escape, cmd, args, env):
 		env['SHLINKFLAGS'][1] = '-shared'
 		env['SHLIBSUFFIX'] = '.so'
 	elif (os.name=="nt"):
-		gcc_path=gcc_path+"/windows-x86_64/bin" #this may be wrong
```

Comment:
For me this path is correct and works out of the box. It's been a long while since I had build issues on Windows. Or have this changed in more recent NDK versions?